pages/contribute.md

```diff
@@ -0,0 +1,135 @@
+
+- **[Google Season of Docs](https://developers.google.com/season-of-docs)** \
+    A program that connects technical writers with open-source projects to improve documentation quality and accessibility.
+
+- **[Outreachy](https://www.outreachy.org/)** \
+    A diversity-focused program offering paid remote internships to individuals underrepresented in tech, providing opportunities to contribute to open-source software.
+
+- **[Station1 Frontiers Fellowship](https://www.station1.org/sff)**  \
+    A unique research and internship program designed for students in science and technology, promoting inclusive and socially impactful research.
+
+- **[CompChemURG](https://www.bindingsites.co.uk/home)**  \
+    A mentoring initiative focused on computational chemistry, supporting undergraduates and early-career researchers in gaining expertise in the field.
+
+Many of our [core developers][] started through these programs &mdash; consider joining and becoming part of the MDAnalysis community!
+
+### GSoC
+
+MDAnalysis has participated in Google Summer of Code for several years, mentoring students on a variety of open-source projects. Explore our past projects:
+
+- [GSoC 2025][] 
+- [GSoC 2024][] 
+- [GSoC 2023][] 
+- [GSoC 2022][] 
+- [GSoC 2021][] 
+- [GSoC 2020][] 
+- [GSoC 2019][] 
+- [GSoC 2018][] 
+- [GSoC 2017][] 
+- [GSoC 2016][] 
+
+## Other Contributions
+
+If you are interested in contributing in other ways, such as improving the website or blog, you can refer to the [README file][] in the [Website Repository][]. While the website is already maintained by the MDAnalysis team, minor fixes and updates are always welcome. 
+
+If you'd like to write a **blog post**, we welcome contributions that document your experience with MDAnalysis—whether it's learning the software, participating in a UGM, or working on a project. Feel free to reach out via the [Community]({{ site.baseurl }}/pages/community/) page for guidance on submitting a post.
+
+[MDAnalysis Project]: https://github.com/MDAnalysis/
+[GitHub discussions]: https://github.com/MDAnalysis/mdanalysis/discussions
+[core developers]: https://github.com/orgs/MDAnalysis/teams/coredevs/members
+[Website Repository]: https://github.com/MDAnalysis/MDAnalysis.github.io
+[README file]: https://github.com/namiroues/MDAnalysis.github.io/blob/master/README.md
+[GSoC 2025]: https://www.mdanalysis.org/2025/02/28/gsoc2025/
+[GSoC 2024]: https://www.mdanalysis.org/2024/02/27/gsoc2024/
+[GSoC 2023]: https://www.mdanalysis.org/2023/02/22/gsoc2023/
+[GSoC 2022]: https://www.mdanalysis.org/2022/03/07/gsoc2022/
+[GSoC 2021]: https://www.mdanalysis.org/2021/03/09/gsoc2021/
+[GSoC 2020]: https://www.mdanalysis.org/2020/02/22/gsoc2020/
+[GSoC 2019]: https://www.mdanalysis.org/2019/03/04/gsoc2019/
+[GSoC 2018]: https://www.mdanalysis.org/2018/02/14/gsoc2018/
+[GSoC 2017]: https://www.mdanalysis.org/2017/03/02/gsoc2017/
+[GSoC 2016]: https://www.mdanalysis.org/2016/04/26/gsoc/
```

Comment:
These are all blog post so please use the `{% post_url FILENAME %}` tag to generate stable URLs (just `git grep post_url` to see examples).